codegen/smithy-ruby-codegen-test/integration-specs/streaming_spec.rb

```diff
@@ -79,7 +79,28 @@ module WhiteLabel
             client.streaming_operation({}, output_stream: output_stream)
           end
         end
+      end
+
+      # TODO: create an operation with @requireLength
```

Comment:
Should we be doing this?

Reply:
Think I forgot to push the latest updates - tests for requireLength should be there.